src/main/java/io/apimatic/core/types/AdditionalProperties.java

```diff
@@ -93,12 +92,7 @@ public void setAdditionalProperty(String key, T value, boolean skipNullValue) {
 
     @Override
     public String toString() {
-        if (properties.isEmpty()) {
-            return "";
-        }
-
-        return properties.entrySet().stream().map(entry -> entry.getKey() + "=" + entry.getValue())
-                .collect(Collectors.joining(", ", ", ", ""));
+        return properties.toString();
```

Comment:
Can you please add a unit test to ensure that this new format stays same after the future releases.
That unit test will also show the expected output of this toString() override.

Reply:
Here, we are using java map toString implementation. I don't think we require unit tests for map toString implementation.

Reply:
Unit test added